src/repositories/labs-repository.js

```diff
@@ -0,0 +1,37 @@
+const config = require('config');
+const mongoose = require('mongoose');
+const Grid = require('gridfs-stream');
+
+const conn = mongoose.createConnection(config.mongo, { useNewUrlParser: true });
```

Comment:
You should not get here without the mongoose connection, you have the index.js file where you load every service before starting the application, if you do not manage the flow of services you can get into cases where someone request your upload api but the connection is down.